.github/workflows/tests-nightly.yml

```
@@ -204,13 +199,77 @@ jobs:
           CYPRESS_grepTags: ${{ env.GREP_TAGS }}
           STRIPE_SECRET_KEY: ${{ secrets.STRIPE_SECRET_KEY }}
           STRIPE_PUBLIC_KEY: ${{ secrets.STRIPE_PUBLIC_KEY }}
+          SPLIT: ${{ strategy.job-total }}
+          SPLIT_INDEX: ${{ strategy.job-index }}
         with:
           install: false
-          parallel: true
-          group: 'UI - Firefox'
-          record: true
-          tag: ${{ steps.get-env-uri.outputs.ENV_URI }},${{ env.GREP_TAGS }}
           browser: firefox
+      - name: Create reports dir
+        continue-on-error: true
+        if: ${{ ! cancelled() }}
+        run: npm run qa:create-artifacts-dirs
+      - name: Move screenshots into reports dir
+        continue-on-error: true
+        if: ${{ ! cancelled() }}
+        run: npm run qa:move-screenshots
+      - name: Upload reports
+        uses: actions/upload-artifact@v3
+        if: always()
+        with:
+          name: report-${{ strategy.job-index }}
+          path: ./cypress/reports
+          retention-days: 5
+          if-no-files-found: ignore
+  manual-triggered-test-complete:
+    needs: [run-tests-in-parallel]
+    if: ${{always() && github.event_name != 'repository_dispatch' && !contains(needs.run-tests-in-parallel.result, 'skipped') && !contains(needs.run-tests-in-parallel.result, 'cancelled')}}
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version-file: ".nvmrc"
+          cache: npm
+      - name: Cache node modules
+        uses: actions/cache@v2
+        env:
+          cache-name: cache-node-modules
+        with:
+          path: ~/.npm
+          key: ${{ runner.os }}-qa-${{ env.cache-name }}-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-qa-${{ env.cache-name }}-
+            ${{ runner.os }}-qa-
+            ${{ runner.os }}-
+      - name: Install Dependencies
+        if: steps.cache-node-modules.outputs.cache-hit != 'true'
+        run: NODE_OPTIONS=--max_old_space_size=4096 npm install
```

Should be immutable:
```suggestion
        run: NODE_OPTIONS=--max_old_space_size=4096 npm ci
```